languages/csharp/exercises/concept/properties/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+public enum Units
+{
+    Kilograms
+    ,Pounds
+}
+public class WeighingMachine
+{
+    private const float POUNDS_PER_KILOGRAM = 2.20462f;
+    private float inputWeight;
+
+    public Units Units { get; set; } = Units.Kilograms;
+    public float InputWeight
+    {
+        get { return inputWeight; }
+        set
+        {
+            if (value < 0)
+            {
+                throw new ArgumentException("weight cannot be negative");
+            }
+
+            inputWeight = value;
+        }
+    }
+
+    public float DisplayWeight
```

Comment:
Maybe an alternative option for the vanity factor would be to have this return a string that described the weight?

Reply:
The vanity factor is there to illustrate the "private get".  I  don't think making this a string helps.

Reply:
Right. Let's see how the "tare weight" works out.

Reply:
TareAdjustment, USDisplayWeight